### ISSUE TITLE
Want one Dense Ore Block and i get Three

### ISSUE THREAD

**Issue by iTimmyLP** [rwtema/DenseOres#72]
![unbenannt](https://cloud.githubusercontent.com/assets/6671741/6997515/2aa60518-dbbe-11e4-91ea-47716a590858.png)
The Black frame is my config,
the red frame is that i want to add as dense
and the light blue is that, what i actually get.

How i can fix that?

**Comment by acters124** [comment]
Contact that mod author and tell him/her that putting a mod inside a mod WILL cause Distress!
Its:
         S:baseBlockTexture=pcl_lc:naquadah_ore_16
Why? because pcl_lc is the one that handles the mod's ores
and if you want to increase the size of the ore's texture from 16x to 32x change that 16 to a 32 and so on.
examples:
32x:
        S:baseBlockTexture=pcl_lc:naquadah_ore_32
128x:
        S:baseBlockTexture=pcl_lc:naquadah_ore_128
128 is its highest texture size for ores...

TL;DR:
        S:baseBlockTexture=pcl_lc:naquadah_ore_16

**Comment by acters124** [comment]
Oh BTW default values for rendertype and retrogenid won't cause problems.
TL;DR:
        I:renderType=0
        I:retroGenID=0
is what I used

**Comment by acters124** [comment]
proof:
http://imgur.com/njYQOKc
